### PR TITLE
add rector rule for all namespaces except shipment related

### PR DESCRIPTION
**Pull request by AdamKasp**
we also should add rector rules for partial shipment and inventory but I am waiting for an updated upgrade file

### CODE REVIEW

File: config/sets/sylius/plus/up-to-plus-modular.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        'Sylius\Plus\SharedKernel\Exception\ResourceNotSupportedException' => 'Sylius\SharedKernel\Exception\ResourceNotSupportedException',
+        'Sylius\Plus\SharedKernel\ResourceChannelCheckerInterface' => 'Sylius\SharedKernel\Contract\ResourceChannelCheckerInterface',
+        'Sylius\Plus\Controller\DashboardController' => 'Sylius\MultiStorePlugin\ChannelAdmin\Infrastructure\Controller\DashboardController',
+    ]);
+
+    $rectorConfig->ruleWithConfiguration(RenameNamespaceRector::class, [
+        'Sylius\Plus\BusinessUnits' => 'Sylius\MultiStorePlugin\BusinessUnits',
+        'Sylius\Plus\ChannelAdmin' => 'Sylius\MultiStorePlugin\ChannelAdmin',
+        'Sylius\Plus\CustomerPools' => 'Sylius\MultiStorePlugin\CustomerPools',
+        'Sylius\Plus\Loyalty' => 'Sylius\Plus\Loyalty',
```
Comment (lchrusciel) [inline config/sets/sylius/plus/up-to-plus-modular.php]: is it correct?

Comment (GSadee) [inline config/sets/sylius/plus/up-to-plus-modular.php]:
```suggestion
        'Sylius\Plus\Loyalty' => 'Sylius\LoyaltyPlugin',
```

**Review by lchrusciel — COMMENTED** [review]
I have mixed feelings about the amount of tests here. We are checking one line from almost 20. But on the other hand, I'm not sure it will provide much value to write tests for each case

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Adam! :tada:

**Comment by AdamKasp** [comment]
> I have mixed feelings about the amount of tests here. We are checking one line from almost 20. But on the other hand, I'm not sure it will provide much value to write tests for each case

Yea, I also had doubts about how well tested it should be, but after a short conversation with @GSadee and @jakubtobiasz  I decided to make only one, why? Because I use here rules from Rector, so I don't want to make a test for external logic, additionally, all test will be testing the same thing (changing namespaces)

**Pull request merged**